pkg/machines/libvirt.es6

```diff
@@ -414,9 +453,53 @@ LIBVIRT_PROVIDER = {
             failHandler: buildFailHandler({ dispatch, name, connectionName, message: _("VM SEND Non-Maskable Interrrupt action failed")}),
             args: ['inject-nmi', name]
         });
+    },
+
+    GET_HYPERVISOR_MAX_VCPU ({ connectionName }) {
+        logDebug(`${this.name}.GET_HYPERVISOR_MAX_VCPU:`);
+        if (connectionName) {
+            return dispatch => spawnVirsh({connectionName,
```

Comment:
Can this be `read-only` call of `virsh`? This will fail on an `ovirt` host otherwise